streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -213,7 +214,9 @@
     ProcessorTopology processorTopology;
     ProcessorTopology globalTopology;
 
+    private final MockConsumer<byte[], byte[]> consumer;
     private final MockProducer<byte[], byte[]> producer;
+    private final TestDriverProducer testDriverProducer;
```

Comment:
Because we make app method in `StreamsProducer` package private but need access to `commit()` we add `TestDriverProducer` to get access.